README.md

```diff
@@ -1,2 +1,6 @@
 # smile-warrior
 This repo contains code used for smile detector for project of Gradient Science Club.
+
+To create dataset for smile-warrior you need to run 'Adjust_dataset' script, which creates it from fer2013.csv data set. As a result you will receive csv file named 'smile_warrior_dataset'.
```

Comment:
I think the comment in that script already says what it does. In DL repos you usually have sth shorter like:
To prepare dataset:
```
python do_stuff.py
```